src/query/api/v1/handler/prometheus/native/list_tags.go

```diff
@@ -25,6 +25,8 @@ import (
 	"net/http"
 	"time"
 
+	"github.com/m3db/m3/src/x/clock"
+
```

Comment:
nit: Remove superfluous endline.

Reply:
Will do; it's odd that the linter didn't fail here